src/omero_rdf/__init__.py

```diff
@@ -250,13 +250,13 @@ def serialize_graph(self):
             {
                 "@id": "./",
                 "@type": "Dataset",
-                "rocrate:license": "https://creativecommons.org/licenses/by/4.0/",
```

Comment:
Should we drop the context statement about then, @sukunis?

Reply:
I just noticed that adaption of context entity is needed